src/page.ts

```diff
@@ -232,6 +232,27 @@ export class Page extends EventTarget {
     return result.value;
   }
 
+  /**
+   * Set page content
+   */
+  async setContent(content: string): Promise<void> {
+    await this.evaluate(
+      (html) => {
+        const { document } = globalThis as unknown as {
+          document: {
+            open: () => void;
+            write: (html: string) => void;
+            close: () => void;
+          };
+        };
```

Comment:
I wasn't too sure how to enable the dom typing just for this block code 😅 
I can change it if you know how to achieve this

Reply:
As far as I am aware, there isn't really a way to achieve this functionality unfortunately. Triple slash directives only seem to be valid at the top level, which is really unfortunate.